components/p3a/rotation_scheduler.cc

```diff
@@ -104,9 +103,8 @@ const char* GetConstellationRotationTimestampPref(MetricLogType log_type) {
     case MetricLogType::kExpress:
       return kLastExpressConstellationRotationTimeStampPref;
     default:
-      NOTREACHED_IN_MIGRATION();
+      NOTREACHED();
   }
-  return nullptr;
```

Comment:
in this file, shouldn't we follow the pattern from the guide
```
// Good:
//
// This switch handles all cases, but enums can technically hold any integer
// value (even if all enum members are enumerated), so the compiler must try to
// handle other cases too. We can avoid dealing with values outside enums by
// using NOTREACHED() while also making sure we hear about it.
switch (my_enum) {
  case A: return 1;
  case B: return 5;
  case C: return 3;
}
NOTREACHED();
```

Reply:
forgot to update, thanks